tests/src/PluginExtendedTest.php

```
@@ -0,0 +1,52 @@
+        $this->plugin->deactivate($composer, $io);
+        $this->assertTrue(true); // Method executes without error
+    }
+
+    public function testUninstall(): void
+    {
+        $composer = $this->createMock(Composer::class);
+        $io = $this->createMock(IOInterface::class);
+
+        // Should not throw any exception
+        $this->plugin->uninstall($composer, $io);
+        $this->assertTrue(true); // Method executes without error
```

PHPStan (2.1.15)

Call to method PHPUnit\Framework\Assert::assertTrue() with true will always evaluate to true.


(method.alreadyNarrowedType)